content/blog/2016/2016-04-07-jenkins-community-survey-results-blog.adoc

```diff
@@ -0,0 +1,105 @@
+with Jenkins.
+
+Back in 2012, at the time of the last community survey, 83% of you felt
+that Jenkins was mission-critical. By 2015, the percentage saying that
+Jenkins was mission-critical was 92%. Additionally, echoing the
+importance of Jenkins, 89% of respondents said their use of Jenkins had
+increased over the last year, while 11% said it had stayed the same. 0%
+said that it had decreased.
+
+The trend in the industry over the last couple of years has been to
+adopt continuous delivery (CD), thus pushing automation further down the
+pipeline – from development all the way into production. Jenkins is
+readily adaptable to this trend, since it is an orchestrator of
+automation processes, but not tied to any specific phase of software
+delivery. With its extensive plugin ecosystem, Jenkins can integrate
+with almost any technology used in any phase of software delivery.
+
+The trend to CD is clearly reflected amongst the community: 59% of
+respondents are using Jenkins for CI, but an additional 30% have
+extended CI into CD and are manually deploying code to production.
+Finally, 11% are practicing continuous deployment – they have extended
+CI to CD and are deploying code automatically into production.
+
+Another trend tied to CD adoption and DevOps culture is the frequent
+deployment of incremental releases to production. Of those utilizing
+continuous delivery practices, 26% are deploying code at least once per
+day. Another 37% are deploying code at least once per week.
+
+In keeping with the move to CD, 30% of survey takers are already using
+the relatively new Pipeline plugin to automate their software delivery
+pipelines. Of those not using the Pipeline plugin, 79% plan to adopt it
+in the next 12 months.
+
+Survey takers multi-task with Jenkins, using it for many different
+activities. 97% of survey takers use it for build – no surprise, since
+that is where Jenkins got its start - but fully 58% now also use it for
+deployment.
+
+image:/images/blog/2016-survey-blog-strongbutler.jpg[image,width=264,height=131]ow
+
+When the 2012 community survey was conducted, container technology
+didn’t even really exist and no one would have known what a “Docker”
+was. Fast-forward four years later and fully 96% of those using Linux
+containers are using Docker. Containerization has been a huge trend and,
+when coupled with automation, helps to also accelerate software delivery
+tremendously. Fortunately, Jenkins plays well with containers and robust
+community plugins exist for both Docker and Kubernetes.
+
+The Jenkins improvements most desired by survey takers were:
+quality/timely bug fixes, a better UI and more documentation/examples.
+Interestingly, Jenkins 2.0 - which is just about to officially launch,
+takes a big step in improving the UI and the new Jenkins.io website
+takes a big step towards providing more and better documentation. Nice
+to see community concerns being addressed!
```

Comment:
I will strike the word "community"  and set as "nice to see concerns being addressed"